otus-02/src/otus_02/homework/pangram.clj

```diff
@@ -1,6 +1,13 @@
 (ns otus-02.homework.pangram
-  (:require [clojure.string :as string]))
+  (:require [clojure.string :as string]
+            [clojure.set :refer [difference]]))
 
+(def ^:private alpha (set "abcdefghijklmnopqrstuvwxyz"))
 
-(defn is-pangram [test-string])
-
+(defn is-pangram
+  [test-string]
+  (->> test-string
+       (string/lower-case)
+       (set)
+       (difference alpha)
+       (empty?)))
```

Comment:
Btw, you convert all the initial string into set. I hope you may find all the alphabet on the first 3 pages of Moby Dick f.e., and it's not needed to read up to the end of the book :)

Reply:
Right. In this case, I assume lazy seq is the best choice.